src/acrcssc/azext_acrcssc/_validators.py

```diff
@@ -78,11 +78,20 @@ def _validate_continuouspatch_config(config):
         raise InvalidArgumentValueError(f"Configuration error: Version {config.get('version', '')} is not supported. Supported versions are {CONTINUOUSPATCH_CONFIG_SUPPORTED_VERSIONS}")
 
 
-def check_continuous_task_exists(cmd, registry):
-    exists = False
-    for task_name in CONTINUOUSPATCH_ALL_TASK_NAMES:
-        exists = exists or _check_task_exists(cmd, registry, task_name)
-    return exists
+# to save on API calls, we offer the option to return the list of tasks
+def check_continuous_task_exists(cmd, registry, task_list=None):
+    exists = True
```

Comment:
Do we need 'exists' here? Can we simply the code like:
try:
        acrtask_client = cf_acr_tasks(cmd.cli_ctx)
        for task_name in CONTINUOUSPATCH_ALL_TASK_NAMES:
            task = get_task(cmd, registry, task_name, acrtask_client)
            if task is None:
               return False
            if task_list is not None and task is not None:
                task_list.append(task)
        return True
    except Exception as exception:
        logger.debug(f"Failed to find task {task_name} from registry {registry.name} : {exception}")
        return False